packages/core/src/context.ts

```diff
@@ -69,3 +70,35 @@ export const composeContext = ({
 export const addHeader = (header: string, body: string) => {
     return body.length > 0 ? `${header ? header + "\n" : header}${body}\n` : "";
 };
+
+/**
+ * Generates a string with random user names populated in a template.
+ *
+ * This function generates a specified number of random user names and populates placeholders
+ * in the provided template with these names. Placeholders in the template should follow the format `{{userX}}`
+ * where `X` is the position of the user (e.g., `{{user1}}`, `{{user2}}`).
+ *
+ * @param {string} params.template - The template string containing placeholders for random user names.
+ * @param {number} params.length - The number of random user names to generate.
+ * @returns {string} The template string with placeholders replaced by random user names.
+ *
+ * @example
+ * // Given a template and a length
+ * const template = "Hello, {{user1}}! Meet {{user2}} and {{user3}}.";
+ * const length = 3;
+ *
+ * // Composing the random user string will result in:
+ * // "Hello, John! Meet Alice and Bob."
+ * const result = composeRandomUser({ template, length });
+ */
```

Comment:
Can this be part of a README somewhere?

Reply:
nothing there belongs in the readme, just looks like an LLM being too verbose. It's all dev facing stuff